### PR TITLE
Added Garmin 18x, Smart6 and Smart7

### PR DESCRIPTION
**Pull request by roni-kreinin**

### CODE REVIEW

File: clearpath_sensors/config/garmin_18x.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+nmea_navsat_driver:
+  ros__parameters:
+    port: "/dev/ACM0"
```
Comment (tonybaltovski) [inline clearpath_sensors/config/garmin_18x.yaml:3]: Should be `/dev/clearpath/gps`

File: clearpath_sensors/config/novatel_smart6.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+nmea_navsat_driver:
+  ros__parameters:
+    port: "/dev/ACM0"
```
Comment (tonybaltovski) [inline clearpath_sensors/config/novatel_smart6.yaml:3]: Should be `/dev/clearpath/gps`

File: clearpath_sensors/config/novatel_smart7.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+nmea_navsat_driver:
+  ros__parameters:
+    port: "/dev/ACM0"
```
Comment (tonybaltovski) [inline clearpath_sensors/config/novatel_smart7.yaml:3]: Should be `/dev/clearpath/gps`

**Review by tonybaltovski — CHANGES_REQUESTED** [review]

File: clearpath_sensors/config/garmin_18x.yaml (hunk 1 above)
Comment (roni-kreinin) [reply clearpath_sensors/config/garmin_18x.yaml:3]: Do we need to add a udev rule for this?

Comment (tonybaltovski) [reply clearpath_sensors/config/garmin_18x.yaml]: It is this rule: https://github.com/clearpathrobotics/clearpath_robot/blob/main/clearpath_robot/debian/udev#L7

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**